@here/harp-lrucache/lib/LRUCache.ts

```diff
@@ -60,21 +60,15 @@ export class LRUCache<Key, Value> {
      */
     private m_oldest: Entry<Key, Value> | null = null;
 
-    /**
-     * The internal cost function
-     */
-    private m_sizeFunction: (v: Value) => number;
-
     /**
      * Creates a new instance of `LRUCache`. The optional sizeFunction can be used
      * fine tune the size required to cache that item.
      *
      * @param cacheCapacity The maximum number of entries to store in the cache.
      * @param sizeFunction A function determining the size per element.
      */
-    constructor(cacheCapacity: number, sizeFunction: (v: Value) => number = () => 1) {
+    constructor(cacheCapacity: number, public sizeFunction: (v: Value) => number = () => 1) {
```

Comment:
Any reason this should be public? At least, make it readonly as I don't think it's a good idea to replace the `sizeFunction` at runtime

Reply:
I can modify it to be a single function for all cases. Good idea.

Reply:
Changed